sources/packages/api/src/services/form/form.service.ts

```diff
@@ -99,19 +104,38 @@ export class FormService {
    * in case the result is anything different from HTTP 200 code.
    */
   private async getUserLogin() {
-    const authRequest = await axios.post(`${this.config.formsUrl}/user/login`, {
-      data: {
-        email: this.config.serviceAccountCredential.userName,
-        password: this.config.serviceAccountCredential.password,
-      },
-    });
+    try {
+      const authRequest = await axios.post(
+        `${this.config.formsUrl}/user/login`,
+        {
+          data: {
+            email: this.config.serviceAccountCredential.userName,
+            password: this.config.serviceAccountCredential.password,
+          },
+        },
+      );
 
-    if (authRequest.status !== HttpStatus.OK) {
-      throw new Error(
-        `Error while retrieving formio authentication token. Error ${authRequest.status}: ${authRequest.statusText}`,
+      if (authRequest.status !== HttpStatus.OK) {
+        throw new Error(
+          `Error while retrieving formio authentication token. Error ${authRequest.status}: ${authRequest.statusText}`,
+        );
+      }
+
+      return authRequest;
+    } catch (excp) {
+      this.logger.error(`Received exception while getting form SA token`);
```

Comment:
This is axios level exception, in case of any http exception axios throw exception not any proper response. You can get error response through `excp.response` and `excp.data`.

Reply:
@andrewsignori-aot I will remove the code in line 118, the catch block is handling all generic error condition.